wyrm/processing.py

```diff
@@ -1838,6 +1838,74 @@ def calculate_classwise_average(dat, classaxis=0):
     return dat.copy(data=data, axes=axes)
 
 
+def calculate_cca(dat_x, dat_y, timeaxis=-2):
+    """Calculate the Canonical Correlation Analysis (CCA) of specified data.
+
+    This method calculates the canonical correlation and the corresponding weights.
```

Comment:
Could you expand the long description just a little bit more? A few lines will be sufficient.